### PR TITLE
FFQ32024

### PR DESCRIPTION
**Pull request by aznrayizzle**

### CODE REVIEW

**Review by petermetz — COMMENTED** [review]
@aznrayizzle Are you intentionally overwriting the Q2 report with the contents of the Q3 report or is it by accident?

**Comment by aznrayizzle** [comment]
Was a mistake. I've readded the Q2 report back.

**Review by jimthematrix — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-FireFly.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+__Updates__
+
+- __Improved Documentation:__ Missing documentation and README files have been added, including updates to doc links.
+- __Multiparty Status API:__ A new API was added for managing multiparty transaction status.
+- __Ethereum Blank Signing Key Support:__ Queries on Ethereum can now allow blank signing keys.
+- __Enhanced Listener Management:__ When re-creating listeners in EVMConnect, EthConnect, or FabConnect, the last event block is now used to prevent duplicate handling.
+- __Namespace Management:__ Fixes related to preventing multiple blockchain plugins from starting on namespace retry.
+- __Websocket Enhancements:__ Websockets are now available as an option to communicate with EVM nodes in addition to RPC calls.
+- __Performance:__ Event indexing optimized and bugs fixed to ensure that events and transactions are received and in the correct order.
+- __Receipt Confirmation:__ Improved the receipt confirmations logic from the chain for EVM
```
Comment (mbrandenburger) [inline project-reports/2024/2024-Q3-Hyperledger-FireFly.md]:
```suggestion
- __Receipt Confirmation:__ Improved the receipt confirmations logic from the chain for EVM.
```

File: project-reports/2024/2024-Q3-Hyperledger-FireFly.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+- Multi-filters per listeners: Added the ability to listen to multiple types of events on the same contract listener.
+- New Listener Type: Block listeners are added to stream receipts of block confirmations in the order that they are confirmed on the blockchain.
+- Improved Documentation: Missing documentation and README files have been added, including updates to doc links.
+- Multiparty Status API: A new API was added for managing multiparty transaction status.
+- Ethereum Blank Signing Key Support: Queries on Ethereum can now allow blank signing keys.
+- Enhanced Listener Management: When re-creating listeners in EVMConnect, EthConnect, or FabConnect, the last event block is now used to prevent duplicate handling.
+- Namespace Management: Fixes related to preventing multiple blockchain plugins from starting on namespace retry.
+- Websocket Enhancements: Websockets are now available as an option to communicate with EVM nodes in addition to RPC calls.
+- Performance: Event indexing optimized and bugs fixed to ensure that events and transactions are received and in the correct order.
+- Receipt Confirmation: Improved the receipt confirmations logic from the chain for EVM
```
Comment (mbrandenburger) [inline project-reports/2024/2024-Q3-Hyperledger-FireFly.md]:
```suggestion
- Receipt Confirmation: Improved the receipt confirmations logic from the chain for EVM.
```

**Review by tkuhrt — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by petermetz — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-FireFly.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+- Namespace Management: Fixes related to preventing multiple blockchain plugins from starting on namespace retry.
+- Websocket Enhancements: Websockets are now available as an option to communicate with EVM nodes in addition to RPC calls.
+- Performance: Event indexing optimized and bugs fixed to ensure that events and transactions are received and in the correct order.
+- Receipt Confirmation: Improved the receipt confirmations logic from the chain for EVM.
+
+For a full list of changes and updates, please visit [the FireFly Github](https://github.com/hyperledger/firefly/releases/tag/v1.3.1).
+
+# Overall Activity in the Past Quarter
+
+The major highlights have been detailed at the beginning of the report. In summary, there has been a lot of activity in the project. New community members continue to discover the project and make contributions. This quarter we have continued to see contribution and activities from outside the set of the original maintainers. A new Cardano Connector has been discussed to be in development by an external party.
```
Comment (arsulegai) [inline project-reports/2024/2024-Q3-Hyperledger-FireFly.md:77]: Would this external party join the maintainers, increase diversity with the Cardano Connector contribution?

**Review by arsulegai — APPROVED** [review]

**Pull request merged**